upgrading/downgrade.adoc

```diff
@@ -40,64 +32,51 @@ installations] of {product-title} and assumes downtime of the entire cluster.
 [[downgrade-verifying-backups]]
 == Verifying Backups
 
-The Ansible playbook used during the
-xref:../upgrading/index.adoc#install-config-upgrading-index[upgrade
-process] should have created a backup of the *_master-config.yaml_* file and the
-etcd data directory. Ensure these exist on your masters and etcd members:
-
+. The Ansible playbook used during the
+xref:../upgrading/index.adoc#install-config-upgrading-index[upgrade process]
+should have created a backup of the *_master-config.yaml_* file. Ensure these
+exist on your masters:
++
 ----
 /etc/origin/master/master-config.yaml.<timestamp>
-/var/lib/etcd/openshift-backup-pre-upgrade-<timestamp>
 /etc/origin/master/scheduler.json.<timestamp>
 ----
 
-Also, back up the *_node-config.yaml_* file on each node (including masters,
-which have the node component on them) with a timestamp:
+. The procedure in xref:automated_upgrades.adoc#preparing-for-an-automated-upgrade[Preparing for an Automated Upgrade]
+instructed you to back up the following files before proceeding with an upgrade
+from {product-title} 3.9 to 3.10; ensure that you still have these files
+available:
++
+----
+/etc/systemd/system/atomic-openshift-master-api.service
+/etc/systemd/system/atomic-openshift-master-controllers.service
+----
 
+. Back up the *_node-config.yaml_* file on each node (including masters, which
+have the node component on them) with a timestamp:
++
```

Comment:
Does the timestamp have to be in any certain format? I'm assuming it's whatever plays nice with YAML and may not need to be mentioned here, but just wondering.

Reply:
Nah, shouldn't matter.